src/components/LightUpChinatown/LightUpChinatownPage.tsx

```diff
@@ -68,6 +70,20 @@ const LightUpChinatownPage = () => {
           <CampaignInfoText color={'#CF6E8A'}>
             {daysUntilEnd} {t('lightUpChinatown.campaignDaysLeft')}
           </CampaignInfoText>
+        </TextContainer>
+        <MapContainer>
+          <Map src={mapImg} />
+          <br></br>
+          <GoalContainer>
+            <GoalPill src={goal1Img}></GoalPill>
+            <GoalText>{t('lightUpChinatown.goal1')}</GoalText>
+          </GoalContainer>
+          <br></br>
+          <br></br>
```

Comment:
Should this spacing be more in the form of margin or padding instead of `<br>`'s? Non-blocking either way